src/main/resources/assets/lambdynlights/lang/it_it.json

```diff
@@ -0,0 +1,19 @@
+{
+  "lambdynlights.menu.light_sources": "Fonti di Luci Dinamiche",
+  "lambdynlights.menu.title": "Impostazioni di LambDynamicLights",
+  "lambdynlights.menu.canvas.1": "Disattivare luci dinamiche di entità è raccomandato con Canvas!",
+  "lambdynlights.menu.canvas.2": "L'illuminazione dinamica con Canvas è pesante sulla performance per il momento!",
+  "lambdynlights.option.block_entities": "Entità blocco",
+  "lambdynlights.option.entities": "Entità",
+  "lambdynlights.option.mode": "Luci Dinamiche",
+  "lambdynlights.option.water_sensitive": "Controllo di sensibilità dell'acqua",
+  "lambdynlights.toast.first_time": "Disattivato di predefinito, controlla impostazioni della mod o di Grafica!",
+  "lambdynlights.tooltip.block_entities": "Attiva l'illuminazione dinamica delle entità blocco.",
+  "lambdynlights.tooltip.creeper_lighting": "Imposta la modalità di illuminazione dinamica dei Creeper.\n- %s disattiva l'illuminazione dinamica dei Creeper.\n- %s imposta una luminosità costante.\n- %s imposta una luminosità dinamica.",
+  "lambdynlights.tooltip.entities": "Attiva l'illuminazione dinamica di entità. Nota: I giocatori sono sempre attivi.",
+  "lambdynlights.tooltip.mode.1": "Attiva le luci dinamiche. Se attivo, un giocatore con una torcia in mano emetterà luce per esempio.",
```

Comment:
Il "per esempio" non suona super naturale e manca un articolo.
"Attiva le luci dinamiche. Se attivo, **per esempio,** un giocatore con una torcia in mano emetterà **della** luce~~per esempio~~."

(Anche questa non mi piace tantissimo nel modo in cui è formulata in inglese, ma questa traduzione dovrebbe essere ugualmente accurata)